scripts/deploy.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+
+import os
+import sys
+from time import time, sleep
+import random
```

Comment:
No, scripts are just currently excluded from formatting I think. I'll submit a separate PR to fix that.